src/plone/restapi/tests/test_documentation.py

```diff
@@ -177,6 +177,8 @@ def setUp(self):
         self.portal = self.layer["portal"]
         self.portal_url = self.portal.absolute_url()
 
+        setattr(self.portal, "_plone.uuid", "SomeUUIDPortal")
```

Comment:
I would generate an actual uuid (once, and then include it as a literal here in the test) so that the documentation shows the expected format.

Reply:
Sounds reasonable.